hack/make-rules/test.sh

```diff
@@ -356,9 +362,9 @@ runTests() {
 
     # Include all coverage reach data in the combined profile, but exclude the
     # 'mode' lines, as there should be only one.
-    for x in `find "${cover_report_dir}" -name "${cover_profile}"`; do
-      cat ${x} | grep -h -v "^mode:" || true
-    done
+    while IFS='' read -r x; do
+      grep -h -v "^mode:" < "${x}" || true
+    done < <(find "${cover_report_dir}" -name "${cover_profile}")
```

Comment:
```
In ./hack/make-rules/test.sh line 359:
    for x in `find "${cover_report_dir}" -name "${cover_profile}"`; do
             ^-- SC2044: For loops over find output are fragile. Use find -exec or a while read loop.
             ^-- SC2006: Use $(...) notation instead of legacy backticked `...`.
```
```
In ./hack/make-rules/test.sh line 360:
      cat ${x} | grep -h -v "^mode:" || true
          ^--^ SC2086: Double quote to prevent globbing and word splitting.
          ^--^ SC2002: Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead.
```